design-toolkit/components/src/components/drawer/index.tsx

```diff
@@ -149,11 +150,12 @@ function DrawerLayout({
 DrawerLayout.displayName = 'Drawer.Layout';
 DrawerLayout.Main = DrawerLayoutMain;
 
+// NOTE: this is used to force the prop to one of these values or undefined
 const tooltipPlacementMap = {
-  left: 'right',
-  right: 'left',
-  top: 'bottom',
-  bottom: 'top',
+  [PLACEMENT.LEFT]: PLACEMENT.LEFT,
+  [PLACEMENT.RIGHT]: PLACEMENT.RIGHT,
+  [PLACEMENT.TOP]: PLACEMENT.TOP,
+  [PLACEMENT.BOTTOM]: PLACEMENT.BOTTOM,
```

Comment:
```suggestion
  [PLACEMENT.LEFT]: PLACEMENT.RIGHT,
  [PLACEMENT.RIGHT]: PLACEMENT.LEFT,
  [PLACEMENT.TOP]: PLACEMENT.BOTTOM,
  [PLACEMENT.BOTTOM]: PLACEMENT.TOP,
```
We want to position the tooltip opposite the drawer position.  The tooltip should be positioned right when the drawer is positioned left.